### ISSUE TITLE
GET notifications: error in running the example

### ISSUE THREAD

**Issue by betefaber** [dojot/dojot#1688]
Error in executing the example URI.

![image](https://user-images.githubusercontent.com/37310063/91899715-ef652980-ec73-11ea-98ba-ddd8f5ac003d.png)

- example URI
```
 curl -X GET http://10.50.11.181:30001/history/notifications/history?subject="user_notification" -H "Authorization: Bearer ${JWT}"  -H 'Content-Type:application/json'
<html>
  <head>
    <title>Internal Server Error</title>
  </head>
  <body>
    <h1><p>Internal Server Error</p></h1>

  </body>
</html>
```
log:
```
Will retrieve notifications
user_notification
[2020-09-01 19:52:16 +0000] [9] [ERROR] Error handling request /notifications/history?subject=user_notification
Traceback (most recent call last):
  File "/usr/src/venv/lib/python3.6/site-packages/gunicorn-19.6.0-py3.6.egg/gunicorn/workers/async.py", line 52, in handle
    self.handle_request(listener_name, req, client, addr)
  File "/usr/src/venv/lib/python3.6/site-packages/gunicorn-19.6.0-py3.6.egg/gunicorn/workers/ggevent.py", line 152, in handle_request
    super(GeventWorker, self).handle_request(*args)
  File "/usr/src/venv/lib/python3.6/site-packages/gunicorn-19.6.0-py3.6.egg/gunicorn/workers/async.py", line 103, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/usr/src/venv/lib/python3.6/site-packages/falcon-1.0.0-py3.6.egg/falcon/api.py", line 189, in __call__
    responder(req, resp, **params)
  File "/usr/src/app/history/api/models.py", line 214, in on_get
    query = NotificationHistory.get_query(filter_query)
  File "/usr/src/app/history/api/models.py", line 230, in get_query
    value = HistoryUtil.model_value(value, HistoryUtil.check_type(value))
  File "/usr/src/app/history/api/models.py", line 102, in model_value
    return int(value)
ValueError: invalid literal for int() with base 10: 'user_notification'
10.244.1.19 - - [01/Sep/2020:19:52:16 +0000] "GET /notifications/history?subject=user_notification HTTP/1.1" 500 - "-" "-"
```

- without optional parameters
```
curl -X GET http://10.50.11.181:30001/history/notifications/history -H "Authorization: Bearer ${JWT}"  -H 'Content-Type:application/json' 
{
  "notifications": [
    {
      "msgID": "5f3cf103-e061-44e1-a9f3-08840383e796",
      "metaAttrsFilter": {
        "prioridade": "baixa",
        "shouldPersist": true
      },
      "message": "teste do n\u00f3 notification",
      "subject": "user_notification",
      "ts": "2020-08-27T17:15:33.761000Z"
    }
  ]
}
```

log:
```
Will retrieve notifications
10.244.1.19 - - [01/Sep/2020:19:51:07 +0000] "GET /notifications/history HTTP/1.1" 200 250 "-" "curl/7.58.0"
```

**Comment by betefaber** [comment]
Fixed in version **v0.4.3**

![image.png](https://images.zenhubusercontent.com/5cd0940469cf6b4ee443403a/4f185adc-9920-49b8-ba1c-a9f0cbbea090)

**Comment by betefaber** [comment]
Fixed in **v0.5.0-rc.1**
